reaktive/src/iosCommonMain/kotlin/com/badoo/reaktive/scheduler/Schedulers.kt

```diff
@@ -0,0 +1,29 @@
+package com.badoo.reaktive.scheduler
+
+/**
+ * Creates a new instance of Main [Scheduler]
+ */
+actual fun createMainScheduler(): Scheduler {
```

Comment:
It's not a good idea to create classes with same names in same packages. E.g. for JVM compilation fails with message "Duplicate JVM class name". Let's put each function into a separate file (like CreateComputationScheduler.kt, CreateIoScheduler.kt, etc.).